### PR TITLE
Add produce to osmcrops.csv

### PR DESCRIPTION
**Pull request by rugk**
and improve it generally, created by @westnordost.

Fixes https://github.com/rugk/crops-parser/issues/36
As a demonstration I also generated one YAML file with it.

### CODE REVIEW

**Comment by westnordost** [comment]
Okay, so as noted in #39, you can replace the nuts with the specific nuts.

**Comment by rugk** [comment]
Your change of pepper to chilli and pipers to peppers is confusing. Should not produce=pepper also belong to pepper_plants and so on?

**Comment by westnordost** [comment]
Oh right, probably. This must be a mistake.

File: osmcrops.csv [hunk 1]
```diff
@@ -1,77 +1,77 @@
...
-Jojoba seed,trees=jojoba_plants,orchard,
-Cranberries,trees=cranberry,orchard,
-Grapefruit (inc. pomelos),trees=grapefruit_trees,orchard,
-Tomatoes,crop=tomatoe,MAYBE building=greenhouse,
+crop name,key (OSM produce value),OSM trees-key value,landuse key,comment
+Apples,apple,apple_trees,orchard,
+Bananas,banana,banana_plants,orchard,
+Coffee,coffee,coffee_plants,orchard,
+Coffee- green,coffee,coffee_plants,orchard,
+Grapes,grape,,vineyard,should retag as landuse=vineyard
```
Comment (rugk) [inline osmcrops.csv]: Hmm, a vineyard could also "produce=wine"… :wink:
Not that easy this "produce" tag either.

File: osmcrops.csv [hunk 2]
```diff
@@ -1,77 +1,77 @@
...
+Tangerines- mandarins- clementines- satsumas,orange,orange_trees,orchard,could also be mapped each one under: trees=mandarin_trees
+???????,lychee,lychee_trees,orchard,
+???????,sand_pear,sand_pear_trees,orchard,
+???????,,eucalyptus_trees,orchard,
+???????,pitaya,pitaya_plants,orchard,
+???????,cherimoya,cherimoya_trees,orchard,
+???????,pomegranate,pomegranate_trees,orchard,
+???????,pecan_nut,pecan_trees,orchard,
+# CURRENTLY MISSING/UNOFFICIAL/PROPOSALS BELOW,,,orchard,
+Chillies and peppers- green,chilli,pepper_plants,orchard,
```
Comment (rugk) [inline osmcrops.csv]: Also note: "and peppers". In the source data this does not only cover chillies. If they are really tagged differently in OSM, we could, however, split one source item into two OSM tags, no problem.

See [lemons/limes](https://github.com/rugk/crops-parser/pull/45/files#diff-7fb40528971b8aef808122c5c09891caR26).

**Review by rugk — COMMENTED** [review]
Everything else looks fine.

**Comment by westnordost** [comment]
Soo, merge this?

File: osmcrops.csv [hunk 3]
```diff
@@ -1,77 +1,77 @@
...
+Chillies and peppers- dry,chilli,pepper_plants,orchard,
```
Comment (westnordost) [inline osmcrops.csv]: I misspelled chili

**Comment by rugk** [comment]
> Soo, merge this?

Please see other comments, first.

File: osmcrops.csv (hunk 3 above)
Comment (rugk) [reply osmcrops.csv]: chilli is possible, at least according to LEO. But with one l seems to be more common. I'll change that.

**Comment by westnordost** [comment]
Which one?

**Comment by rugk** [comment]
https://github.com/rugk/crops-parser/pull/45#discussion_r131002048 and https://github.com/rugk/crops-parser/pull/45#discussion_r131002746

I said "everything **else** looks fine" in my review with these two comments.

**Comment by rugk** [comment]
I actually found out: piper and petter [are synonyms](https://en.wikipedia.org/wiki/Piper_(plant)). So what to use?

Note: The scientific name Piper and the common name "pepper"

File: osmcrops.csv (hunk 1 above)
Comment (westnordost) [reply osmcrops.csv]: The farmland itself not, perhaps the whole wine-making company inclusive the farmyard.

File: osmcrops.csv (hunk 2 above)
Comment (westnordost) [reply osmcrops.csv]: ok

**Comment by westnordost** [comment]
Pepper

File: osmcrops.csv (hunk 1 above)
Comment (rugk) [reply osmcrops.csv]: Okay.

File: osmcrops.csv (hunk 2 above)
Comment (rugk) [reply osmcrops.csv]: Just FYI: Of course, chillies are also peppers.

**Comment by rugk** [comment]
LGTM, @westnordost?

And BTW: It sucks somehow to have to change things in a PR, which is not mine. So next time, I'd prefer a real PR. Just saying… It's easier that way. :smiley:

**Comment by westnordost** [comment]
All good, thanks!

**Pull request merged**